### DISCUSSION TITLE
What's the proper way to do a "in" filter if django only takes last value?

### DISCUSSION THREAD

**Discussion by mhamid3d** [carltongibson/django-filter#1468] [Q&A]
When querying with multiple values, as such:

`api/products/?category__in=Office?category__in=Kitchen?category__in=Bedroom`

Before cleanup the querydict will be:
`category__in=['Office', 'Kitchen', 'Bedroom']`

but after cleaning up the data, Django only takes the last value:
`category__in=['Bedroom']`

What is the proper way to do this then?

**Comment by carltongibson** [comment]
Are you sure you're using a multiple choice filter? 

See e.g. https://django-filter.readthedocs.io/en/stable/ref/filters.html#modelmultiplechoicefilter
